src/Forms/Shared/Samples/Symbology/SimpleRenderers/SimpleRenderers.xaml.cs

```diff
@@ -49,6 +46,12 @@ private void Initialize()
             // Use the two points farthest apart to create an envelope
             Envelope initialEnvelope = new Envelope(oldFaithfullPoint, plumeGeyserPoint);
 
+            // Use the envelope to define the map views visible area (include some padding around the extent)
+            myMap.InitialViewpoint = new Viewpoint(initialEnvelope, 100);
```

Comment:
For some reason the MapView starts off rotated. This does NOT affect usability of the sample — no changes needed.

I'll have to investigate why this happens on XF-UWP and not on native WPF/UWP. Might be related to that viewpoint bug I mentioned on RasterLayer...